backends-velox/src/test/scala/org/apache/spark/sql/execution/VeloxParquetWriteSuite.scala

```diff
@@ -70,43 +73,38 @@ class VeloxParquetWriteSuite extends WholeStageTransformerSuite {
 
   test("test ctas") {
     withTable("velox_ctas") {
-      intercept[UnsupportedOperationException] {
-        spark
-          .range(100)
-          .toDF("id")
-          .write
-          .format("parquet")
-          .saveAsTable("velox_ctas")
-      }
+      spark
+        .range(100)
+        .toDF("id")
+        .createOrReplaceTempView("ctas_temp")
+      val df = spark.sql("CREATE TABLE velox_ctas USING PARQUET AS SELECT * FROM ctas_temp")
+      Assert.assertTrue(FallbackUtil.isFallback(df.queryExecution.executedPlan))
     }
   }
 
   test("test parquet dynamic partition write") {
     withTempPath {
       f =>
-        intercept[UnsupportedOperationException] {
-          spark
-            .range(100)
-            .selectExpr("id as c1", "id % 7 as p")
-            .write
-            .format("parquet")
-            .partitionBy("p")
-            .save(f.getCanonicalPath)
-        }
+        val path = f.getCanonicalPath
+        spark
+          .range(100)
+          .selectExpr("id as c1", "id % 7 as p")
+          .createOrReplaceTempView("temp")
+        val df = spark.sql(s"INSERT OVERWRITE DIRECTORY '$path' USING PARQUET SELECT * FROM temp")
```

Comment:
Here change seems loss the semantic partition by  and can not verify that the dynamiac partition write